### ISSUE TITLE
Consider adding "bad indentation" (from pylint)

### ISSUE THREAD

**Issue by NeilGirdhar** [astral-sh/ruff#6428]
Consider:
```python
class X:
        x: int
```
Pylint says:
```
************* Module a
a.py:2:0: W0311: Bad indentation. Found 8 spaces, expected 4 (bad-indentation)
```
but Ruff says nothing.

Could we consider adding a bad indentation rule?  That might also necessitate adding a setting for "indentation" (like line-length).

**Comment by 153957** [comment]
This is also tracked by https://github.com/astral-sh/ruff/issues/970

**Comment by NeilGirdhar** [comment]
@153957 Ah, my mistake.  I searched for issues, but forgot about that one.  Should we close this then?

**Comment by zanieb** [comment]
Thanks!
